apps/jira/jira-app/src/components/App/Config.tsx

```diff
@@ -47,6 +47,9 @@ export default class Config extends React.Component<Props, State> {
       this.loadEditorInterfaces(),
     ]);
 
+    console.log('app sdk>>>', app);
```

Comment:
If we're going to add logs like this, I'd vote that we prefix these msgs with `Jira Project >>>>` and `Slack project >>>>`. This way things are very clear when the customer reports the error back to us

Reply:
really good call 👍 have adjusted!